client/src/components/admin/organizationTable.js

```diff
@@ -85,12 +72,10 @@ export default class OrganizationList extends Component {
     })
   }
 
-  renderOrganizationTable() {
+  renderTable() {
     const headerValues = [
```

Comment:
Should we have another way to uniquely identify a program/organization with the same name in this table?

Reply:
I guess we could show 'id'? I'm not sure if that's very helpful though

Reply:
I thought this was the motivation behind adding the name of any and all parent organizations to the program payload.. I just did that recently in order to support the differentiation between programs for this very page. For organizations, if my duplication prevention strategy is any good, which I feel pretty good about it, then we will never have the same name organization at the same location. This means that the first part of the address could be used to differentiate between organizations if we want.

Reply:
Okay, I'll add those in there. We just get a list of org names right? Is it possible to add address in there as well? This is definitely an edge case but if the program is a part of two orgs with the same name, there wouldn't be any way to differentiate them.

Reply:
Corresponding addresses could be included as well, but only in their attribute form (separated address_line_1 and address_line_2). This means that you would have to concatenate them into a more meaningful, combined address line on the FE. If that's feasible, I'll add them to the payload, otherwise we can just deal with having that edge case as a risk for our mvp.

Reply:
That would work fine on the frontend

Reply:
Okay cool, so the way that the query returns its result, it will be a list of lists like this:
```
"parent_organization_names": [
                    [
                        "Xcel Energy",
                        "52323 Maureen Mountains",
                        "Apt. 628"
                    ],
                    ...
]
```
Just so that you know what you'll be working with. I can try to add in some more logic to reformat it during the serialization process if you think this will be a hassle.